sos/report/plugins/__init__.py

```diff
@@ -2982,6 +2982,10 @@ def add_journal(self, units=None, boot=None, since=None, until=None,
         self._add_cmd_output(cmd=journal_cmd, timeout=timeout,
                              sizelimit=log_size, pred=pred, tags=tags,
                              priority=priority)
+        self.add_cmd_tags({
+            "journalctl --no-pager": "journal_all",
+            "journalctl --no-pager --boot": "journal_since_boot"
+        })
```

Comment:
These should already be applied:

```
{
                            "command": "journalctl",
                            "parameters": [
                                "--no-pager",
                                "",
                                "--boot",
                                ""
                            ],
                            "exec": "journalctl --no-pager  --boot ",
                            "filepath": "sos_commands/logs/journalctl_--no-pager_--boot",
                            "truncated": false,
                            "return_code": 0,
                            "priority": 10,
                            "start_time": 1675189264.6049192,
                            "end_time": 1675189265.145264,
                            "run_time": 0.5403447151184082,
                            "tags": [
                                "journal_since_boot"
                            ]
                        },
```

and

```
{
                            "command": "journalctl",
                            "parameters": [
                                "--no-pager",
                                ""
                            ],
                            "exec": "journalctl --no-pager ",
                            "filepath": "sos_commands/logs/journalctl_--no-pager",
                            "truncated": false,
                            "return_code": 0,
                            "priority": 100,
                            "start_time": 1675189266.1241925,
                            "end_time": 1675189267.3110363,
                            "run_time": 1.1868438720703125,
                            "tags": [
                                "journal_full"
                            ]
                        }
```

Reply:
Hmm let me look into that again, the bottom one is off it's journal_full but the spec name is journal_all.

Reply:
Ah! I guess I went cross-eyed there.

If it's `journal_all`, then I'd say we should just make the `tags` argument for that one a list, since it's already broken out into its own `add_journal()` call.

Reply:
Updated and removed the journal_since_boot and fixed the journal_all.